editor/src/document/movement_handler.rs

```diff
@@ -29,6 +29,7 @@ pub enum MovementMessage {
 	WheelCanvasZoom,
 	SetCanvasRotation(f64),
 	ZoomCanvasToFitAll,
+	TranslateCanvas((f64, f64)),
```

Comment:
This could probably also just use a DVec2 as an argument because we don't need to expose a FFI

Reply:
Done